sdks/cpp/lite/examples/start_here/device.start_here.json

```diff
@@ -36,15 +36,15 @@
             "value": {"string_value": "Hello, World!"}
         },
         "count": {
-          "type": "INT32",
-          "value": { "int32_value": 1234 },
-          "name": { "display_strings": { "en": "Counter" } }
+            "type": "INT32",
+            "value": { "int32_value": 1234 },
+            "name": { "display_strings": { "en": "Counter" } }
         },
         "gain": {
-          "type": "FLOAT32",
-          "value": { "float32_value": 0.707 },
-          "name": { "display_strings": { "en": "Audio Gain" } },
-          "widget": "fader"
+            "type": "FLOAT32",
```

Comment:
please correct indentation (it's too deep) for this and next 3 lines.

install the extension called "Prettier" and use it to format JSON.